openpathsampling/test.py

```diff
@@ -0,0 +1 @@
+print("codecov should scream about this")
```

Comment:
```suggestion
def fake_func():
    print("codecov should scream about this")
```

You need to import the file somewhere -- either in the main package or in a `test_test.py` test module. Coverage doesn't know about files unless they're imported (for example, none of the OPS visualization stuff is even included in unit tests, because it has to be imported separately.)

Also, check the .coveragerc to make sure that we don't exclude anything with "test" in it from coverage.